test/e2e/lib/framework/serverApps/intake.ts

```diff
@@ -2,17 +2,22 @@ import { createInflate } from 'zlib'
 import connectBusboy from 'connect-busboy'
 import express from 'express'
 
-import { SegmentFile, SerssionReplayCall } from '../../types/serverEvents'
+import cors from 'cors'
+import { SegmentFile, SessionReplayCall } from '../../types/serverEvents'
 import { EventRegistry } from '../eventsRegistry'
 
-export function createIntakeServerApp(events: EventRegistry) {
+export function createIntakeServerApp(serverEvents: EventRegistry, bridgeEvents: EventRegistry) {
   const app = express()
 
+  app.use(cors())
```

Comment:
Weirdly, using `XMLHttpRequest` in the e2e bridge makes CORS errors but not sendBeacon. However to have an easy implementation and to be compatible with IE, I stick with XMLHttpRequest and added `cors()` to the e2e intake server.
In any case this configuration is closer to real life scenario.

Reply:
wrong header with xhr maybe?
https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#simple_requests

Reply:
Yes, probably. However, I used the same implementation as the SDK request fallback which doesn't pass headers. Wdyt, should I change the e2e xhr bridge implementation or are we good to go with `cors()`?

Reply:
IMO `cors()` is fine